File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.py
def solution(my_string, indices):
    #큰 인덱스로 문자열 정렬 > 배열 순회해서 삭제 > 배열을 리스트로 만들어서 pop()사용
    indices.sort(reverse=True)
    answer = list(my_string)
        
    for i in indices:
        answer.pop(i)
    
    return ''.join(answer)

File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.java
import java.util.Arrays;
class Solution {
    public String solution(String my_string, int[] indices) {
        Arrays.sort(indices);
        StringBuffer answer = new StringBuffer(my_string);
        
        // 큰 인덱스부터 삭제
        for(int i=indices.length-1;i>=0;i--){
            answer.deleteCharAt(indices[i]);
        }
        
        return answer.toString();
    }
}